pkgs/build-support/vm/windows/default.nix

```diff
@@ -22,7 +22,7 @@ in {
   runInWindowsVM = drv: let
     newDrv = drv.override {
       stdenv = drv.stdenv.override {
-        shell = "/bin/sh";
+        shell = "${stdenv.shell}";
```

Comment:
This one looks a bit odd to me, are you sure it is not deliberately using `/bin/sh` here? In any case, the override as it is in the new version seems redundant.